content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+By contrasting these pairs, the authors surface the content-related features of a quote that make it memorable, and factor out the context in which the quotes appear, context which otherwise plays an important role in the memorability rating.
+After checking that human subjects can identify which quote in the pair is memorable (they do so with an average 78% success rate), the authors show that memorable quotes, on average, use less frequent vocabulary, more frequent grammatical categories (POS tags), and more general constructions (fewer 3^rd^ person pronouns, more indefinite articles, etc.) that make them more adaptable to changing contexts (each of these measures, taken individually, partitions the quote pairs into two subsets containing about 40% vs. 60% of the whole set).
+Cancelling out context effects to develop content-related features has become a widely used approach, with adaptations ranging from the identification of linguistic markers of politeness in online content [@danescu-niculescu-mizil_computational_2013] to the measurement of attractiveness of famous quotes [@acerbi_did_2017].
+In a study reminiscent of @hall_effect_1950, @acerbi_did_2017 compared the relative strength of content and presentational context in a sample of famous quotes that participants had to rate for attractiveness.
+The authors compared conditions where quotes were presented alone, versus presented with random attribution to more or less famous personalities, or versus presented with a random popularity score.
+They found that such minimal context has little effect if any at all:
+attribution, famous or not, bears no effect on the attractiveness of a quote, and popularity has little.
+@althoff_how_2014 also opened the study of context versus content to relational variables, by showing how social status and presentational features (such as showing a strong need) can affect the success of requests on Reddit.
+
+A third related stream of research focuses more specifically on influence in social networks, and its links with attention:
+what network effects trigger the diffusion of a particular meme or piece of information?
+Among the micro-processes involved in the spread of information in networks, what is the role of influence across connected nodes?
+@bakshy_social_2009 investigated the question of social influence by examining information cascades in Second Life.
+Information cascades, where a comparatively small initial event triggers large scale diffusion, are a well-known phenomenon in social networks, and their size distribution is well modelled by peer-pressure threshold models which link the cascade behaviours to the topology of the network in which they occur [@watts_simple_2002; @ruan_kinetics_2015].
+@bakshy_social_2009 thus tracked the spread of *assets
```

Comment:
### Camille
Au passage, le dual-channel vs. fixed-channel ne bénéficie jamais vraiment d'une explicitation.

Reply:
Removed this section. Done.